mapbox/libjava-services/src/test/java/com/mapbox/services/api/staticimage/v1/StaticMarkerAnnotationTest.java

```diff
@@ -65,8 +65,76 @@ public void requireMarkerLon() throws ServicesException {
     new StaticMarkerAnnotation.Builder().setName(Constants.PIN_SMALL).setLat(2.0).build();
   }
 
+  @Test(expected = ServicesException.class)
```

Comment:
IMO just testing that an exception is thrown when an url is not valid is 👌
If more explicitness needed, what about creating a new specific exception?
BTW, is that explicitness really needed in this case?